airbyte-webapp/src/views/Connection/CatalogTree/FieldRow.tsx

```diff
@@ -35,21 +36,16 @@ const RadiobuttonContainer = styled.div<{ depth?: number }>`
   padding-right: ${({ depth }) => (depth ? depth * 38 : 0)}px;
 `;
 
-const FieldRowInner: React.FC<FieldRowProps> = (props) => {
-  const { name: fieldName, onPrimaryKeyChange, onCursorChange } = props;
-  const handlePkChange = useCallback(
-    () => onPrimaryKeyChange(fieldName.split(".")),
-    [fieldName, onPrimaryKeyChange]
-  );
-
-  const handleCursorChange = useCallback(
-    () => onCursorChange(fieldName.split(".")),
-    [fieldName, onCursorChange]
-  );
+const FieldRowInner: React.FC<FieldRowProps> = ({
+  onPrimaryKeyChange,
+  onCursorChange,
+  path,
+  ...props
+}) => {
   return (
     <>
       <FirstCell ellipsis depth={props.depth} flex={1.5}>
-        <NameContainer depth={props.depth} title={props.name}>
+        <NameContainer depth={props.depth} title={path.join(".")}>
```

Comment:
Only 1 comment - as we have name here, we could use `props.name` instead as it will be already joined via `.`

Reply:
I agree, fixed.